docs/installationguide.md

```diff
@@ -87,6 +87,8 @@ IoT Agent. The following attributes are accepted:
 * **exchange**: Exchange in the AMQP broker
 * **queue**: Queue in the AMQP broker
 * **durable**: durable queue flag (default is `false`).
+* **retries**: Number of AMQP connection errror retries (default is 5).
+* **retryTime**: Time between AMQP connection retries (default is 5'').
```

Comment:
`5''`

Typo?

Reply:
5'' == 5 seconds

Reply:
We typically use the style "5 seconds" along all documentation. However, we can also use 5'' (personally, I find it a bit confusing, but it's fine).